Source/player.cpp

```diff
@@ -398,12 +398,14 @@ void StartWalk(int pnum, Displacement vel, Direction dir, bool pmWillBeCalled)
 void SetPlayerGPtrs(const char *path, std::unique_ptr<byte[]> &data, std::array<std::optional<CelSprite>, 8> &anim, int width)
 {
 	data = nullptr;
+#ifndef RUN_TESTS
 	data = LoadFileInMem(path);
 
 	for (int i = 0; i < 8; i++) {
 		byte *pCelStart = CelGetFrame(data.get(), i);
 		anim[i].emplace(pCelStart, width);
 	}
+#endif

```

Comment:
Not a fan of test exceptions, what happens currently with this, and does this PR change that?

Reply:
This only worked before by accident, I think.
The LoadFileInMem function previously returned an empty array in test mode, which was also invalid here because of `GetCelFrame`.

Reply:
We should probably see if we can run the test with the shareware data, ill try to make a mental note of this.